public/docs/_examples/template-syntax/dart/lib/app_component.dart

```diff
@@ -0,0 +1,122 @@
+library template_syntax.app_component;
+
+import 'dart:html';
+import 'dart:convert';
+
+import 'package:angular2/angular2.dart';
+import 'package:template_syntax/hero.dart';
+import 'package:template_syntax/hero_detail_component.dart';
+import 'package:template_syntax/click_me_component.dart';
+import 'package:template_syntax/loop_back_component.dart';
+import 'package:template_syntax/key_up_components.dart';
+import 'package:template_syntax/little_hero.dart';
+
+List<Hero> _MockHeroes = [
+  new Hero('Hercules',
+      lastName: 'Son of Zeus',
+      birthdate: new DateTime(1970, 1, 25),
+      url: 'http://www.imdb.com/title/tt0065832/',
+      rate: 325),
+  new Hero('eenie', lastName: 'toe'),
+  new Hero('Meanie', lastName: 'Toe'),
+  new Hero('Miny', lastName: 'Toe'),
+  new Hero('Moe', lastName: 'Toe')
+];
+
+enum _Color { Red, Green, Blue }
```

Comment:
That is a great question! It's in the [plunkr example](http://plnkr.co/edit/mr63T5?p=info) at the bottom, but scanning through the the actual docs I don't see a reference to it anywhere. 
@kwalrath do we want to scrub it?